rotkehlchen/exchanges/exchange.py

```diff
@@ -507,3 +509,19 @@ def query_history_with_callbacks(
             )
         except RemoteError as e:
             fail_callback(str(e))
+
+    def send_unknown_asset_message(
+            self,
+            msg_aggregator: 'MessagesAggregator',
+            asset_identifier: str,
+            details: str,
+    ) -> None:
```

Comment:
Add a docstring, explainign what the function does, what its arguments are and what errors if any it can raise.